src/app/ApplicationTemplate.Shared.Views/Content/Diagnostics/HttpDebuggerView.xaml.cs

```diff
@@ -24,27 +12,28 @@ public HttpDebuggerView()
 	{
 		this.InitializeComponent();
 
-		var view = ApplicationView.GetForCurrentView();
-		TrySetDetailsScrollViewerMaxHeight(view);
-		view.VisibleBoundsChanged += OnVisibleBoundsChanged;
+		var currentWindow = App.Instance.CurrentWindow;
+		TrySetDetailsScrollViewerMaxHeight(currentWindow);
+		currentWindow.SizeChanged += OnVisibleBoundsChanged;
```

Comment:
That's not the same event.

Reply:
Will have to investigate, needed it to compile for now.